x/proof/keeper/keeper.go

```diff
@@ -63,10 +63,19 @@ func NewKeeper(
 
 	ringClient, err := rings.NewRingClient(ringClientDeps)
 	if err != nil {
-		return Keeper{}, err
+		panic(err)
 	}
 
-	k := Keeper{
+	pubKeyClientDeps := depinject.Supply(
+		accountQuerier,
+	)
+
+	pubKeyClient, err := pubkeyclient.NewPubKeyClient(pubKeyClientDeps)
```

Comment:
`pubKeyClientDeps` could either be in-lined, or substituted for `replayClientDeps`.

Reply:
You mean substituded for `ringClientDeps`?

Reply:
Yes, my mistake. :facepalm: